src/App.js

```diff
@@ -149,8 +157,11 @@ const App = () => {
                         path="/pagination"
                         element={<PagePagination books={bigBookArray} />}
                     />
+                        <Route path='/my-profile' element={<ProfilePage myProfile={!!sessionObject}/>}/>
```

Comment:
Could you please remove myProfile property as now ProfilePage will always show only my profile?